### ISSUE TITLE
Some columns not imported (Bologna Persons)

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#1510]
(Some?) values from residence_id and social_relation1_value_label in [Bologna Persons](https://docs.google.com/spreadsheets/d/1W8YTiKBZZOi_682guwH7N9aEX_qINi9D3qBM8u96Fhk/edit#gid=0)  have not been imported - there is just the proptype, but no propvalue.
E.g. P0002 should have wife and location of residence ([detail](https://dissinet.cz/apps/inkvisitor-data-import/#selectedDetail=583d5b76-a143-42c1-aa6f-3d469fff84b0&detail=583d5b76-a143-42c1-aa6f-3d469fff84b0)).

![obrazek](https://user-images.githubusercontent.com/55400794/213368664-56bc1f55-d45f-4357-bac6-00f0b6ca060a.png)

**Comment by tomaham** [comment]
I know how this one emerged. This is a result of behind-the-scenes stitching of different versions of the import. I have played heavily with Bologna and I made a mistake in syncing the right tables together in certain moment.

Easy to fix. But I will wait for other bugs.

@BTW all of the locations or people props and metaprops connections in Bologna will be messed up because of this. The same source of the problem.
